sbnci/Modules/JobConfigurations/pfpslicevalidation_job_sbnd.fcl

```diff
@@ -1,25 +1,22 @@
-#include "services_sbnd.fcl"
+#include "simulationservices_sbnd.fcl"
 #include "backtrackerservice.fcl"
 #include "particleinventoryservice.fcl"
-#include "simulationservices_sbnd.fcl"
-#include "services_sbnd.fcl"
-#include "sam_sbnd.fcl"
+
 #include "pfpvalidation.fcl"
 
-process_name: ana
+process_name: AnaCI
+
 services:
 {
-  TFileService: { fileName: "pfpSliceValidationHists.root" }
   @table::sbnd_services
```

Comment:
would having the full sbnd simulation services remove the need to include the backtracker here?

Reply:
I don't believe so but I'm happy to be proved wrong [This](https://github.com/SBNSoftware/sbndcode/blob/develop/sbndcode/LArSoftConfigurations/simulationservices_sbnd.fcl#L97-L105) is said table.

Reply:
Have tested this with `fhicl-dump` and actually running the fcs with `lar`, these services need introducing separately as above.